packages/contracts-bedrock/scripts/libraries/L2GenesisHelpers.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+/// @title L2GenesisHelpers
+/// @notice Contains constants and helper methods that are commonly used for generating and testing L2 genesis.
+library L2GenesisHelpers {
+    uint256 constant PRECOMPILE_COUNT = 256;
+    uint256 constant PROXY_COUNT = 2048;
```

Comment:
i believe its internal by default, would be nice to have the internal keyword tho, we dont want any public getters on the library or else it will break our Go tooling